CONTRIBUTING.md

```diff
@@ -70,6 +70,32 @@ change or improvement:
 git clone https://github.com/<your_github_username>/mappymatch.git
 ```
 
+It's advisable to [create and activate a virtual environment](https://docs.python.org/3/library/venv.html) before beginning development.
+
+```
+python3 -m venv /path/to/new/virtual/environment
+
+source /path/to/new/virtual/environment/bin/activate
+```
+
+Change directory to the mappymatch folder:
+
+```
+cd path/to/mappymatch   # from here, your path to mappymatch is likely .\mappymatch\
+```
+
+Install the dependencies and development dependencies
+
+```
+pip install .[dev]
```

Comment:
perhaps we should modify this to include the editable flag for dev environments: `pip install -e ".[dev]"`

Reply:
I think this is correct.

Reply:
Yes, sorry, that's what I meant. Your suggestion to use the `-e` flag is correct.

Reply:
Ahh apologies for misunderstanding!

Reply:
CONTRIBUTING.md was removed as redundant in the meantime.